airbyte-integrations/connectors/source-sftp-bulk/integration_tests/integration_test.py

```diff
@@ -76,33 +104,19 @@ def config_fixture(docker_client):
 
 
 @pytest.fixture(name="config_pk", scope="session")
-def config_fixture_pk(docker_client):
+def config_fixture_pk(docker_client, docker_ip, public_key_path, private_key, path_to_mount_to_sftp):
 
     with socket() as s:
         s.bind(("", 0))
         available_port = s.getsockname()[1]
 
-    ssh_path = TMP_FOLDER + "/ssh"
-    dir_path = os.getcwd() + "/integration_tests"
-
-    if os.path.exists(ssh_path):
-        shutil.rmtree(ssh_path)
-
-    os.makedirs(ssh_path)
-
-    pk, pubk = generate_ssh_keys()
-
-    pub_key_path = ssh_path + "/id_rsa.pub"
-    with open(pub_key_path, "w") as f:
-        f.write(pubk)
-
     config = {
-        "host": "localhost",
+        "host": docker_ip,
```

Comment:
This is the slight networking tweak:
In `airbyte-ci` the docker host is not localhost different as we're in a `dind` context. The client should target the `dockerd` hostname to connect to the container. The `pytest-docker` plugin I installed exposes a convenient `docker_ip` fixture that will return the correct hostname. This fixture will return the correct hostname whether the execution context is with a custom `DOCKER_HOST` or with a "normal" local docker host: integration test should pass both in airbyte-ci or by running them directly from pytest.